lib/remote_helper.py

```diff
@@ -313,8 +314,14 @@ def enable_root_login(cls, host, keyname, infrastructure, is_verbose):
       # logged-in user, so log in as that user to enable root login.
       if infrastructure == "gce":
         AppScaleLogger.log("Root login not enabled - enabling it now.")
-        cls.ssh(host, keyname, 'sudo cp ~/.ssh/authorized_keys /root/.ssh/',
+        temp_file = tempfile.NamedTemporaryFile()
+        cls.ssh(host, keyname, "sudo sort -u ~/.ssh/authorized_keys " \
+          "/root/.ssh/authorized_keys -o {0}".format(temp_file.name),
           is_verbose, user=getpass.getuser())
+        cls.ssh(host, keyname, "sudo sed -n '/.*Please login/d; " \
+          "w/root/.ssh/authorized_keys' {0}".format(temp_file.name),
+          is_verbose, user=getpass.getuser())
```

Comment:
Would it make sense to make a single ssh call with `&&` separating the commands?